### PR TITLE
Make product tests reports available on host machine

### PR DESCRIPTION
**Pull request by wendigo**
This PR will enable reporting product tests executions status back to Github checks.

### CODE REVIEW

File: presto-product-tests-launcher/src/main/java/io/prestosql/tests/product/launcher/cli/TestRun.java [hunk 1]
```diff
@@ -121,6 +130,7 @@ public Execution(EnvironmentFactory environmentFactory, PathResolver pathResolve
             this.testJar = requireNonNull(testRunOptions.testJar, "testOptions.testJar is null");
             this.testArguments = ImmutableList.copyOf(requireNonNull(testRunOptions.testArguments, "testOptions.testArguments is null"));
             this.environment = requireNonNull(testRunOptions.environment, "testRunOptions.environment is null");
+            this.reportsDir = requireNonNull(testRunOptions, "restRunOptions is null").reportsDir;
```
Comment (aalbu) [inline presto-product-tests-launcher/src/main/java/io/prestosql/tests/product/launcher/cli/TestRun.java:133]:
```suggestion
            this.reportsDir = requireNonNull(testRunOptions.reportsDir, "testRunOptions.reportsDir is null");
```

File: presto-product-tests-launcher/src/main/java/io/prestosql/tests/product/launcher/cli/TestRun.java [hunk 2]
```diff
@@ -219,10 +263,10 @@ private void runTests(Environment environment)
 
                 InspectContainerResponse containerInfo = container.getCurrentContainerInfo();
                 ContainerState containerState = containerInfo.getState();
-                Integer exitCode = containerState.getExitCode();
+                Long exitCode = containerState.getExitCodeLong();
```
Comment (aalbu) [inline presto-product-tests-launcher/src/main/java/io/prestosql/tests/product/launcher/cli/TestRun.java:266]: Why was this needed?

File: presto-product-tests-launcher/src/main/java/io/prestosql/tests/product/launcher/cli/TestRun.java [hunk 3]
```diff
@@ -193,12 +204,45 @@ private Environment getEnvironment()
...
+                                .addAll(reportsDirOptions())
                                 .build().toArray(new String[0]));
             });
 
             return environment.build();
         }
 
+        private Iterable<? extends String> reportsDirOptions()
+        {
+            if (Strings.isNullOrEmpty(reportsDir)) {
```
Comment (kokosing) [inline presto-product-tests-launcher/src/main/java/io/prestosql/tests/product/launcher/cli/TestRun.java:216]: static import

**Comment by aalbu** [comment]
Can you look into the CI failures?

File: presto-product-tests-launcher/src/main/java/io/prestosql/tests/product/launcher/cli/TestRun.java (hunk 2 above)
Comment (wendigo) [reply presto-product-tests-launcher/src/main/java/io/prestosql/tests/product/launcher/cli/TestRun.java:266]: No. I've just seen a deprecated method use so I replaced it with non-deprecated one :)

**Comment by wendigo** [comment]
> Can you look into the CI failures?

Yeah. Forgot slashes at the end of command :) Fixed and passed CI

**Comment by kokosing** [comment]
Merged, thanks!
